core/pubnub_helper.c

```diff
@@ -65,6 +65,9 @@ char const* pubnub_res_2_string(enum pubnub_res e)
     case PNR_INVALID_PARAMETERS: return "Invalid function parameters";
     case PNR_ERROR_ON_SERVER: return "Server reported an error";
     case PNR_AUTHENTICATION_FAILED: return "Proxy authentication failed";
+    case PNR_ENTITY_API_INVALID_PARAM: return "Entity API invalid parameter";
```

Comment:
Should be `Objects API`

Reply:
Sure, we can do a "global search and replace" :)